### PR TITLE
Add PWA Page

### PR DESCRIPTION
**Pull request by bshahid331**
The content is taken from here: https://forum.flow.com/t/build-a-walletless-mobile-app-pwa/5400

From design:

Screenshot 2023-11-27 at 2 05 36 PM (1)

### CODE REVIEW

**Review by gregsantos — APPROVED** [review]
Looks good to me. Only question I have is if we should replace the create-react-app template with instruction to install/use the CLI and scaffold

File: docs/build/guides/mobile/walletless-pwa.md [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+```
+
+### **Scripts/Transactions with Flow**
+
+This example shows how to interact with the Flow blockchain using FCL and Magic for authorization.
+
+- **fcl.send**: A function provided by FCL to send transactions or scripts to the Flow blockchain.
+- **AUTHORIZATION_FUNCTION**: The authorization function provided by Magic for signing transactions.
+
+```bash
```
Comment (nialexsan) [inline docs/build/guides/mobile/walletless-pwa.md:187]:
```suggestion
```js
```

File: docs/build/guides/mobile/walletless-pwa.md [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+
+Now we can unlock the real power of Flow. Lets say you have another Flow account and you want to link the “magic” account as a child account so that you can take full custody of whatever is in the magic account you can do this via Hybird Custody.
+
+You can view the hybrid custody repo and contracts here: https://github.com/onflow/hybrid-custody
+
+We will maintain two accounts within the app. The child(magic) account form earlier and new non custodial FCL flow account. I won’t go over how to log in with FCL here and use it but you can do the normal process to obtain the parent account.
+
+One you have the parent account and child(magic) account logged in you can link the account by using the following transaction.
+
+```jsx
```
Comment (nialexsan) [inline docs/build/guides/mobile/walletless-pwa.md:226]:
```suggestion
```cadence
```

File: docs/build/guides/mobile/walletless-pwa.md [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+              manager.addAccount(cap: cap)
+          }
+      }
+```
+
+**Note:** For the sake of this example, well use some pre defined factory and filter implementations. You can find them on the repo but on testnet we can use 0x1055970ee34ef4dc and 0xe2664be06bb0fe62 for the factory and filter address respectively. 0x1055970ee34ef4dc provides NFT capabilities and 0xe2664be06bb0fe62 which is the AllowAllFilter. These generalized implementations likely cover most use cases, but you'll want to weigh the decision to use them according to your risk tolerance and specific scenario
+
+Now, for viewing all parent accounts linked to a child account and removing a linked account, you can follow similar patterns, using Cadence scripts and transactions as required.
+
+```jsx
```
Comment (nialexsan) [inline docs/build/guides/mobile/walletless-pwa.md:305]:
```suggestion
```cadence
```

File: docs/build/guides/mobile/walletless-pwa.md [hunk 4]
```diff
@@ -0,0 +1,407 @@
...
+      
+              let manager = parentAcct.borrow<&HybridCustody.Manager>(from: HybridCustody.ManagerStoragePath)
+                  ?? panic("manager no found")
+      
+              manager.addAccount(cap: cap)
+          }
+      }
+```
+
+**Note:** For the sake of this example, well use some pre defined factory and filter implementations. You can find them on the repo but on testnet we can use 0x1055970ee34ef4dc and 0xe2664be06bb0fe62 for the factory and filter address respectively. 0x1055970ee34ef4dc provides NFT capabilities and 0xe2664be06bb0fe62 which is the AllowAllFilter. These generalized implementations likely cover most use cases, but you'll want to weigh the decision to use them according to your risk tolerance and specific scenario
```
Comment (nialexsan) [inline docs/build/guides/mobile/walletless-pwa.md:301]:
```suggestion
:::note

For the sake of this example, well use some pre defined factory and filter implementations. You can find them on the repo but on `testnet` we can use `0x1055970ee34ef4dc` and `0xe2664be06bb0fe62` for the factory and filter address respectively. `0x1055970ee34ef4dc` provides NFT capabilities and `0xe2664be06bb0fe62` which is the `AllowAllFilter`. These generalized implementations likely cover most use cases, but you'll want to weigh the decision to use them according to your risk tolerance and specific scenario

:::
```

**Review by gregsantos — APPROVED** [review]

**Pull request merged**